src/app/page.tsx

```diff
@@ -139,6 +140,7 @@ const Home = () => {
             {sections.map((i) => (
               <Section key={`section-${i.sortOrder}`} section={i} />
             ))}
+            {carousel.images && <Carousel carousel={carousel} />}
```

Comment:
Empty array is truthy, so we need to use length.  If it's null when there's no carousel entry we should update the interface.